jobs/integr8ly/pds-install.yaml

```diff
@@ -162,6 +188,27 @@
                                     """
                                 }
                             }
+                            if (INSTALL_BACKUPS.toString() == 'true') {
+                                String backupsNamespace = 'openshift-integreatly-backups';
+                                String secretName = 's3-credentials';
+                                String obtainSecretsCommand = 'sudo oc get secrets -n ' + backupsNamespace
+
+                                def projects = sh(script: 'sudo oc projects', returnStdout: true)
+                                if (!projects.contains(backupsNamespace)) {
+                                    sh """
+                                        sudo oc create ns ${backupsNamespace}
+                                    """
+                                }
+                                
+                                def secrets = sh(script: obtainSecretsCommand, returnStdout: true)
+                                if (!secrets.contains(secretName)) {
+                                    writeYaml file: secretFileName, data: secretContent
+                                    sh """
+                                        sudo oc create -f ${secretFileName}
+                                        rm ${secretFileName}
+                                    """
+                                }
+                            }
```

Comment:
Wouldn't be easier just to try to create the secret (with ` sudo oc create -f ${secretFileName}`) and optionally catch error and log some message in case of already existing secret?

Reply:
Probably would be easier, not sure if cleaner though. I don't have any strong preference

Reply:
If I got it right the TEST_SUITE_REPOSITORY and TEST_SUITE_BRANCH are only used for secret creation. It seems that `curl -k <rawGitLabUrlToSecretFile>` is sufficient to get the `secretContent` (see https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/trepel-pipeline-playground/42/console). You just need to run it from `cirhos_rhel7`, not bastion. So you don't need those params at all.

Reply:
It is probably ok on public GH repo, however on GItLab you need to provide your token to get file this way

Reply:
Not sure if I understand. I think this is the simplest way to get the `secetContent`:
```
String secretContent = sh (returnStdout: true, script: 'curl -k https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe/raw/development/resources/QE-AWS-SECRET.yml')
println secretContent
```

No need to clone the GitLab repo. But this piece of code needs to be (not sure) executed from `cirhos_rhel7`, not sure if that would work from bastion.

Reply:
I think you are right. Will update it

Reply:
@trepel mind having another look?